pages/getting_started/configure.rst

```diff
@@ -0,0 +1,127 @@
+*********************
+Initial Configuration
+*********************
+
+Once the application is installed, there are a few items that must be configured before Graylog may be started for the first time. Both the Graylog ``server.conf`` and Elasticsearch ``elasticsearch.yml`` files are configuration files that contain key details needed for intial configuration. 
+
+This guide will provide you with the essential settings to get Graylog up and running. There are many other important settings in these files and we encourage you to review them once you are up and running. For more details, please see :ref:`server/.conf`
+
+.. note:: If you are using the virtual appliance, please skip this section and go directly to :ref:`connect`
+
+server.conf
```

Comment:
I'm not happy having the double content in the docs - But I did not know a better solution currently.

We need to remember in the future during edits of the server.conf description that parts are mirrored in this page.

Reply:
I agree completely, but I don't have a better solution either.